Lesson_7_DirectoryAndThreads/Program.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using Spectre.Console;
+
+class Program
+{
+    static int copiedFilesCount = 0;
+    static object locker = new object();
+
+    static void Main(string[] args)
+    {
+        Stopwatch stopWatch = new Stopwatch();
+
+        var selectOption = AnsiConsole.Prompt(
+            new SelectionPrompt<string>()
+                .Title("Choose an option. Do you want to manually specify paths or use the default system?")
+                .PageSize(3)
+                .AddChoices(new[] {
+                    "Manually specify paths", "Default system" }));
```

Comment:
Use constants

Reply:
You created a static field with a string array. But this doesn't solve the issue. The issue here is that you use a literal string in your code. The same string is used several times in your code and if you want to change it - you have to find all usages and update them. But you can just use string constants and for updating it - you have to update it in a single place - and your code becomes less error-prone.